staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2973,6 +2973,12 @@ type PodSpec struct {
 	// Optional: Defaults to true.
 	// +optional
 	EnableServiceLinks *bool `json:"enableServiceLinks,omitempty" protobuf:"varint,30,opt,name=enableServiceLinks"`
+	// List of user-initiated ephemeral containers to run in this pod.
+	// This field is alpha-level and is only honored by servers that enable the EphemeralContainers feature.
```

Comment:
Describe the rules of how containers can change.

Reply:
You should also describe how someone modifies these (i.e. use this other resource)